app/views/News.js

```diff
@@ -3,11 +3,11 @@ import {
   ActivityIndicator,
   BackHandler,
   Dimensions,
+  ScrollView,
   StyleSheet,
   View,
 } from 'react-native';
 import LinearGradient from 'react-native-linear-gradient';
-import Carousel from 'react-native-snap-carousel';
```

Comment:
If this is the only place where Carousel is used, remove it from package.json too

Reply:
Sure thing!